galsim/phase_screens.py

```diff
@@ -41,7 +41,7 @@ class AtmosphericScreen(object):
     @param altitude      Altitude of phase screen in km.  This is with respect to the telescope, not
                          sea-level.  [default: 0.0]
     @param time_step     Interval to use when advancing the screen in time in seconds.
-                         [default: 0.03]
+                         [default: 0.025]
```

Comment:
I don't actually prefer either one.  :)  Maybe add a sentence to the doc saying that 0.025 is fairly arbitrary, although probably in the right ball park for most purposes.  For careful studies, one would want to check that the results do not change significantly when going to smaller time steps.  Something like that.